src/ralph/data_importer/widgets.py

```diff
@@ -135,11 +136,16 @@ class ImportedForeignKeyWidget(widgets.ForeignKeyWidget):
     def clean(self, value):
         result = None
         if value:
-            content_type, imported_obj = get_imported_obj(self.model, value)
-            if imported_obj:
-                result = self.model.objects.filter(
-                    pk=int(imported_obj.object_pk)
-                ).first()
+            if settings.IMPORT_DATA_FROM_OLD_RALPH:
+                content_type, imported_obj = get_imported_obj(
+                    self.model, value
+                )
+                if imported_obj:
+                    result = self.model.objects.filter(
+                        pk=int(imported_obj.object_pk)
```

Comment:
At least on mysql this is working:
```
In [2]: Warehouse.objects.get(pk=1)
Out[2]: <Warehouse: DC1>

In [3]: Warehouse.objects.get(pk="1")
Out[3]: <Warehouse: DC1>
```
We should check how django handles str and int here (maybe all ints are mapped to str?)